<?xml version="1.0"?>
<launch>
    <group ns="aura">
    <node pkg="pc_mapping" type="pc_slam" name="aura_mapping" output="screen" respawn="true">
        <param name="map_frame" type="string" value="/map"/>
        <param name="odom_frame" type="string" value="/odom"/>
        <param name="map_topic" type="string" value="base_map"/>
        <param name="pc_topic" type="string" value="/camera/depth/points"/>
        <param name="hz" type="int" value="10"/>
        <param name="height" type="int" value="4"/>
        <param name="width" type="int" value="7"/>
        <param name="resolution" type="double" value="0.1"/>
        <param name="initial_x" type="int" value="-1"/>
        <param name="initial_y" type="int" value="-2"/>
        <param name="min_tolerance" type="double" value="0.05"/>
        <param name="max_tolerance" type="double" value="1"/>
    </node>
    </group>
</launch>
